lib/datapacktypes/omniscript.js

```diff
@@ -154,11 +154,11 @@ OmniScript.prototype.afterActivationSuccess = async function (inputMap) {
             return;
         }
 
-        await this.compileOSLWC(jobInfo, omniScriptId,omniScriptKey);
-        
+        await this.compileOmniScriptLwc(jobInfo, omniScriptId, omniScriptKey);
```

Comment:
It can't replace the old way. It should still use old way if not at the right version for this.